### PR TITLE
Add LexasCMS items to Ember Times issue 157

### PR DESCRIPTION
**Pull request by mike183**
## What it does
Adds blurbs for the LexasCMS items of Ember Times issue 157

### CODE REVIEW

File: source/2020-07-24-the-ember-times-issue-157.md [hunk 1]
```diff
@@ -14,47 +14,41 @@ READMORE
...
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [New Ember Data addon for LexasCMS released ⚡️](https://github.com/LexasCMS/ember-data-lexascms)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Michael Donaldson (@mike183)](https://www.github.com/mike183) has published the new **ember-data-lexascms** addon for easily integrating LexasCMS with any Ember.js application.
```
Comment (amyrlam) [inline source/2020-07-24-the-ember-times-issue-157.md:28]:
```suggestion
<!--alex ignore easy-->
[Michael Donaldson (@mike183)](https://www.github.com/mike183) has published the new **ember-data-lexascms** addon for easily integrating LexasCMS with any Ember.js application.
```

This is from the alex linter, this change will fix the Travis CI build

**Comment by amyrlam** [comment]
@mike183 thanks a lot for this! I'm thinking it would be better as one article with subtopics, but can take a closer look and add a suggestion for what I mean later today

File: source/2020-07-24-the-ember-times-issue-157.md [hunk 2]
```diff
@@ -14,47 +14,41 @@ READMORE
...
+## [LexasCMS: Introducing the JSON:API Content Delivery API 🎉](https://www.lexascms.com/blog/introducing-the-jsonapi-content-delivery-api/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Have you been looking to integrate a headless CMS with your Ember.js application?
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+LexasCMS recently shipped a new **JSON:API compliant content delivery API** which aims to make this easier then ever before.
```
Comment (abhilashlr) [inline source/2020-07-24-the-ember-times-issue-157.md]:
```suggestion
LexasCMS recently shipped a new **JSON:API compliant content delivery API** which aims to make this integration easier than ever before.
```

**Review by abhilashlr — COMMENTED** [review]
Thank you for the amazing write up @mike183 👍 I’ve left a minor change. Also I feel the same way as @amyrlam has suggested to have this write up  as one article containing sub topics.

**Comment by amyrlam** [comment]
@mike183 echo-ing what @abhilashlr said, this is an awesome writeup, thank you so much! 💯

I'm having some trouble making a commit on your fork. I think it may be easiest if you resolve or comment on the suggestions here, and then I could merge this in, and make some minor copyedits in another branch upstream on `ember-blog`. Let me know!

**Comment by mike183** [comment]
@amyrlam @abhilashlr Thanks for the reviews!

I've pulled in your suggested changes, made a couple of small copy tweaks and also took a stab at combining them into a single article.

Let me know if you'd like me to make anymore changes 🙂

**Comment by amyrlam** [comment]
@mike183 this looks great, thanks. Let's get this merged. If we make further edits before Friday, I'll @ you on GitHub. Is it ok if I add your name as an author to the issue?

**Comment by mike183** [comment]
@amyrlam Great, thanks! Sure, adding me as an author is fine 🙂

**Pull request merged**